tests/test_verilog/test_bind.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_bind():
-    RTL4 = RTL.generate(4)
```

Comment:
we don't use `import rtl_monitor` anywhere in this file right?

Reply:
We need to import it so that the bind call is done (the pattern is import triggers the bind, so by including a monitor file, the bind logic is added, this reflects how the pattern is done in verilog, since the bind is added by including addition files).